rcsb/utils/dictionary/DictMethodCommonUtils.py

```diff
@@ -4047,6 +4049,124 @@ def getNeighborInfo(self, dataContainer, distLimit=5.0, targetModelId="1"):
         logger.info("Completed %s at %s (%.4f seconds)", dataContainer.getName(), time.strftime("%Y %m %d %H:%M:%S", time.localtime()), time.time() - startTime)
         return rD
 
+    def getCompModelDb2L(self, dataContainer):
+        """Get list of comp model database_ids from database_2
+
+        Args:
+            dataContainer (object): mmif.api.DataContainer object instance
+
+        Returns:
+            compModelDb2L = ["AlphaFoldDB", ...]
+
+        """
+        try:
```

Comment:
I think you should initiate the `compModelDb2L` variable here, so that it can return something if it fails. Also, at the end of the `if` statement, might be good to check that `compModelDb2L` isn't empty, and if it is, log an error message.

Reply:
Done.